lib/modules/platform/scm-manager/readme.md

```diff
@@ -0,0 +1,21 @@
+# SCM-Manager
+
+Renovate supports the [SCM-Manager](https://scm-manager.org) platform.
+
+## Authentication
+
+1. Create an API Key for your technical Renovate user in SCM-Manager
+1. The technical user _must_ have a valid name and email address
+1. Put the API key in the `RENOVATE_TOKEN` environment variable, so that Renovate can use it
+
+You must set the `platform` config option to `scm-manager` in your Renovate config file.
```

Comment:
I would like us to link to the docs. I know you removed the link because you could not figure out the correct format. Let's try again. 🙃 

@viceice can you suggest the correct link to the `platform` self-hosted config option docs?

https://docs.renovatebot.com/self-hosted-configuration/#platform

Reply:
https://github.com/renovatebot/renovate/blob/f39568d8a809a3b170ecb812f21e38c1689e1ce0/lib/modules/platform/azure/readme.md#L128

```md
[`platform`](../../../configuration-options.md#platform)
```